src/chatkit.ts

```diff
@@ -394,6 +424,36 @@ export default class Chatkit {
     });
   }
 
+  private updatePermissionsForRole(
+    roleName: string,
+    scope: string,
+    permissionsToadd: Array<string>,
+    permissionsToRemove: Array<string>,
+  ): Promise<any> {
+    if (len(permissionsToadd) == 0 && len(permissionsToRemove) == 0) {
+      throw new Error(`Either permissionsToAdd or permissionsToRemove is required`)
+    }
+
+    var body: object = {}
+    if (len(permissionsToadd) > 0) {
```

Comment:
Not Go 😛

`permissionsToAdd.length`

`permissionsToadd` -> `permissionsToAdd`

Reply:
Been writing Go for far too long..